src/emulator/auth/operations.ts

```diff
@@ -658,7 +689,10 @@ function getProjects(
   };
 }
 
-function getRecaptchaParams(): Schemas["GoogleCloudIdentitytoolkitV1GetRecaptchaParamResponse"] {
+function getRecaptchaParams(
```

Comment:
Noticed that no tests currently exist for `getProjects` and `getRecaptchaParams` - not sure if this was intentionally left out, clarification on this front would be appreciated!

cc: @yuchenshi